src/modules/developer-guide/pages/documentation/Conventions.1.adoc

```diff
@@ -0,0 +1,260 @@
+
+The bellow list is aggregated from https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/[AsciiDoc], shortened and adjusted to our needs. - !!! - ???
+
+Please, do your best to *follow the conventions* listed below when contributing to the documentation.
+
+We are quite strict in having *1 sentence per line* to have a nicer version history of the document in production.
+
+== Composition, formation, layout
+When composing any text it is important to pay attention its formation and layout, especially so when we talk about documents *viewed via electronic devices*.
+
+It is essential to produce a document that is *easy to understand* and *stable* from both the perspective of contents and the visual perspective.
+Consistency is crucial.
+
+It may be good to:
+
+.Visually group
+Make sure you group components that fit together, i.e., discussions of the same concept, concept and an example, subpoints and tips...
+
+[NOTE]
+If something is not relevant in a section, move it elsewhere.
+It is better to add a section than to compromise on coherency and fluidity of the document. 
+
+.Use formatting to break up monotone text
+* Use *bold*, *italic*, *links*, *inline code blocks*, and other formatting to point out crucial points in a paragraph.
+* *Do not go overboard*, as highlighting excessively results in the decrease of attentitivenes to a highlighted sections.
+
+== Lists
+
+You can use lists to provide the document with *additional structure*, to *outline* the essential points, display a *sequence* etc.
+Try to keep lists *short*, as they can make the document harder to read.
+
+.Punctuation:
+* If a list item is a *proper sentence*, start with a *capital letter*, and end with a *period*.
+* If a list item is *not* a *proper sentence*, start with a *lowercase letter*, and (optionally) end with a *comma*.
+In case you choose to use commas, start the first item with a capital letter end the last one with a period. 
+* If the entire list is one sentece broken down into clauses for each point, make sure to start the first list item with a capital letter, finish up the point with a comma, continue with a lowercase letter in the following point, finish it up with a comma and so on.
+Wrap the list up with a period.
+* Do not mix sentence patterns when producing a list.
+
+.Example of a proper sentence; the outline of our documents:
+* The *end-user guide* talks about things relevant to the end-users.
+* The *integrator guide* talks about creating new things based on the {PRODUCT_NAME}.
+* The *developer guide* helps the core developers keep track on how to maintain different features.
+
+=== Ordered lists
+
+List items in an *ordered list* should *start with a dot* (`. ...content here...`) instead of an actual number (it gets assigned at compile-time).
+Please, see the below example.
+
+[source,adoc]
+----
+.Here we can see:
+. list item 1,
+. list item 2.
+----
+
+=== Unordered lists
```

Comment:
@nikabrlec can you add that multiple `*` should be used when you are indenting the line item; eg:

```
* item
** sub item
*** sub sub item
```